src/Mobs/Spider.cpp

```diff
@@ -43,9 +43,15 @@ void cSpider::EventSeePlayer(cEntity * a_Entity)
 		return;
```

Comment:
But what if the server is under heavy load and it takes, say, 3 seconds until the chunk is processed and lighted? During that time, each spider in the chunk will add the chunk to the queue 60 times. Is my understanding of the threads correct? I'm assuming lots of ticks can occur before a chunk is lighted and removed from the queue.

> how many spiders will there be in the chunk?
It depends. I'd say 0 - 5 on average.